examples/user.yaml

```
@@ -0,0 +1,660 @@
+        type: future
+  
+  - &tenants_terms
+    name: terms
+    fields:
+      - name: id
+        type: id
+      - name: country_id
+        type: *countries
+      - name: version
+        type: integer
+      - name: text 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)